colossalai/zero/gemini/chunk/chunk.py

```diff
@@ -90,9 +91,11 @@ def __init__(
         self.chunk_size = chunk_size
         self.utilized_size = 0
 
-        self.torch_pg = process_group
+        self.torch_pg = zero_group
         self.pg_size = dist.get_world_size(self.torch_pg)
         self.pg_rank = dist.get_rank(self.torch_pg)
+        self.extra_dp_group = extra_dp_group
+        self.extra_dp_size = dist.get_world_size(self.extra_dp_group)
```

Comment:
If `extra_dp_group` is None, then `extra_dp_size` is global world size, which is wrong

Reply:
updated.